src/wallet/wallet.cpp

```diff
@@ -1773,6 +1773,7 @@ CBlockIndex* CWallet::ScanForWalletTransactions(CBlockIndex* pindexStart, CBlock
         }
         if (pindex && fAbortRescan) {
             LogPrintf("Rescan aborted at block %d. Progress=%f\n", pindex->nHeight, gvp);
+            ret = pindex;
```

Comment:
Please revert this change. This breaks other rpcs that use `ScanForWalletTransactions` directly

Reply:
Done